sdk/servicebus/service-bus/src/models.ts

```diff
@@ -267,4 +267,8 @@ export interface PeekMessagesOptions extends OperationOptionsBase {
    * The sequence number to start peeking messages from (inclusive).
    */
   fromSequenceNumber?: Long;
+  /**
+   * (Experimental for diagnostic purpose) Specifies whether to omit the body when peeking messages.
```

Comment:
same comment about listing the default value.

Reply:
Can we mark this as `@beta`?

Reply:
Done!